src/platform/silabs/wifi/WifiInterfaceAbstraction.h

```diff
@@ -28,10 +28,12 @@
 #include <stdbool.h>
 
 /* LwIP includes. */
+#ifndef SLI_SI91X_OFFLOAD_NETWORK_STACK
```

Comment:
nit. open ticket to check if we need this anymore

Reply:
Okay will do